<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FightingWidget</class>
 <widget class="QWidget" name="FightingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>371</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButtonSkill0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: palegoldenrod;
    border-width: 2px;
    border-color: darkkhaki;
    border-style: solid;
    border-radius: 5;
    padding: 3px;
    min-width: 9ex;
    min-height: 2.5ex;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
    </property>
    <property name="text">
     <string>技能1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSkill1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: palegoldenrod;
    border-width: 2px;
    border-color: darkkhaki;
    border-style: solid;
    border-radius: 5;
    padding: 3px;
    min-width: 9ex;
    min-height: 2.5ex;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
    </property>
    <property name="text">
     <string>技能2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSkill2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>80</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: palegoldenrod;
    border-width: 2px;
    border-color: darkkhaki;
    border-style: solid;
    border-radius: 5;
    padding: 3px;
    min-width: 9ex;
    min-height: 2.5ex;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
    </property>
    <property name="text">
     <string>技能3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSkill3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: palegoldenrod;
    border-width: 2px;
    border-color: darkkhaki;
    border-style: solid;
    border-radius: 5;
    padding: 3px;
    min-width: 9ex;
    min-height: 2.5ex;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
    </property>
    <property name="text">
     <string>技能4</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelPicA">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;

background-color: rgb(255, 255, 255);
border-color: rgb(16, 16, 16);
border: 2px solid rgb(0,0,0);
border-radius : 1px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPicB">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;

background-color: rgb(255, 255, 255);
border-color: rgb(16, 16, 16);
border: 2px solid rgb(0,0,0);
border-radius : 1px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarHPA">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
	color: rgb(41, 255, 59);
    border: 2px solid grey;
    border-radius: 5px;
}

QProgressBar::chunk {
    background-color: #37ff70
}

QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarHPB">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
}

QProgressBar::chunk {
    background-color:  #37ff70
}

QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frameDiscard">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>491</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;
background-color: rgb(255, 255, 255);


border-color: rgb(16, 16, 16);
border: 2px solid rgb(0,0,0);
border-radius : 15px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;
font: 25 14pt &quot;Microsoft YaHei UI Light&quot;;
background-color:rgb(255, 255, 255);


border-color: rgb(16, 16, 16);
border: 0px solid rgb(0,0,0);
border-radius : 15px;</string>
    </property>
    <property name="text">
     <string>战败，请选择要丢弃的小精灵</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDiscard">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>189</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
	font: 75 14pt &quot;Segoe Print&quot;;
    background-color: rgb(208, 208, 208);
  	 border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 17px;
}

QPushButton:hover {
   background-color: rgb(165, 165, 165);
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
    </property>
    <property name="text">
     <string>丢弃</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;Microsoft YaHei&quot;;</string>
    </property>
    <property name="text">
     <string>小精灵1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;Microsoft YaHei&quot;;</string>
    </property>
    <property name="text">
     <string>小精灵2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;Microsoft YaHei&quot;;</string>
    </property>
    <property name="text">
     <string>小精灵3</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowserDisplay">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHpAText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHpBText">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>159</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
